lis3dh/registers.go

```diff
@@ -0,0 +1,77 @@
+package lis3dh
+
+// Constants/addresses used for I2C.
+
+// The I2C addresses which this device listens to.
+const (
+	Address0 = 0x18 // SA0 is low
+	Address1 = 0x19 // SA0 is high
+)
+
+// Registers. Names, addresses and comments copied from the datasheet.
+const (
+	WHO_AM_I             = 0x0F
+	LIS3DH_REG_STATUS1   = 0x07
```

Comment:
Is there any reason you used the `LIS3DH_` prefix? This is the lis3dh package, so the prefix is unnecessary (see "effective Go").

Reply:
Removed.